invenio_github/ext.py

```diff
@@ -52,6 +52,19 @@ def release_api_class(self):
         assert issubclass(cls, GitHubRelease)
         return cls
 
+    @cached_property
+    def release_error_handlers(self):
+        """Github Release error handlers."""
+        error_handlers = current_app.config.get('GITHUB_ERROR_HANDLERS', {})
```

Comment:
```suggestion
        error_handlers = current_app.config.get('GITHUB_ERROR_HANDLERS', [])
```

`GITHUB_ERROR_HANDLERS = None` should be included in `config.py` with a docstring on what it does.